src/api/VoteMonitor.Api.Core/Handlers/PaginatedQueryHandlerBase.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Linq;
+
+namespace VoteMonitor.Api.Core.Handlers
+{
+    public class PaginatedQueryHandlerBase
```

Comment:
Can we please move this to `VoteMonitor.Api.Core.Extensions` and rename `PaginatedQueryHandlerBase` to `QueryExtension`. 
This is my final comment i sear ! :)

Reply:
@idormenco , done, please, take a look! :)